forgevtt-module.js

```diff
@@ -1791,40 +1786,65 @@ class ForgeAPI {
 }
 
 class ForgeCompatibility {
+
+    /**
+     * The global isNewerVersion will be removed in v14, so we need a utility function to alias to whichever is available.
+     * @param {string} version The version to check
+     * @param {string} target The version to check against
+     * @returns {boolean} True when the version is newer than the target, false otherwise
+     */
+    static _isNewerVersion(version, target) {
+        try {
+            return foundry.utils.isNewerVersion(version, target);
+        } catch (_) {
+            return isNewerVersion(version, target);
+        }
+    }
+
+    static get isNewerVersion() {
+        return ForgeCompatibility._isNewerVersion
+    }
+
     static get TextureLoader() {
-        // TextureLoader is not available in Foundry v11 and up
-        if (ForgeVTT.utils.isNewerVersion(ForgeVTT.foundryVersion, "12")) {
+        if (ForgeCompatibility.isNewerVersion(ForgeVTT.foundryVersion, "12")) {
             return foundry.utils.TextureLoader;
         }
         return window.TextureLoader;
     }
 
     static get ModuleManagement() {
-        if (ForgeVTT.utils.isNewerVersion(ForgeVTT.foundryVersion, "12")) {
+        if (ForgeCompatibility.isNewerVersion(ForgeVTT.foundryVersion, "12")) {
             return foundry.appv1.apps.ModuleManagement;
         }
         return window.ModuleManagement;
     }
 
     static get Module() {
-        if (ForgeVTT.utils.isNewerVersion(ForgeVTT.foundryVersion, "12")) {
+        if (ForgeCompatibility.isNewerVersion(ForgeVTT.foundryVersion, "12")) {
             return foundry.packages.Module;
         }
         return window.Module;
     }
 
     static get MergeObject() {
```

Comment:
Most of the other static members using pascal case, I get, since they're mapping classes. This maps a function, though. Maybe we rename it to `mergeObject` to match what Foundry uses?

Reply:
Agreed!